src/Microsoft.Azure.WebJobs.Logging/Internal/TimeBucket.cs

```diff
@@ -27,5 +29,63 @@ public static long ConvertToBucket(DateTime dt)
             }
             return min;
         }
+
+        // Used to compute epoch. 
+        const long ticksBaseline = 633663648000000000L;
+        const long ticksPerMonth = 26785144663987L;
+        
```

Comment:
actually - we don't need them anymore in the latest iteration. So I'll just deleted them. 

---
In reply to: [82881677](https://github.com/Azure/azure-webjobs-sdk/pull/862#discussion_r82881677) [](ancestors = 82881677)